balboa-client-jms/src/main/java/com/blist/metrics/impl/queue/MetricJmsQueue.java

```diff
@@ -64,7 +64,15 @@ private MetricJmsQueue(ConnectionFactory factory, String queueName) throws JMSEx
         this.factory = factory; // badness happens if it's GC'd
         Connection connection = factory.createConnection();
         realInstance = new MetricJmsQueueNotSingleton(connection, queueName);
-        realInstance.start();
+    }
+
+    @Override
+    public void close() throws Exception {
```

Comment:
So you mean there is no super.close to chain to? It's abstract or from an interface?